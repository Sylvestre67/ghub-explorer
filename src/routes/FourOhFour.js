import React from 'react';
import PropTypes from 'prop-types';

import { withStyles } from '@material-ui/core';

import Typography from '@material-ui/core/Typography';

const styles = theme => ({
	root: {
		textAlign: 'center',
		padding: `${theme.spacing.unit * 3}px 0`,
	},
});

export function FourOhFour(props) {
	const { classes } = props;

	return (
		<div className={classes.root}>
			<Typography variant="h6" color="error">
				Not Found
			</Typography>
			<Typography>The Page you requested cannot be found.</Typography>
		</div>
	);
}

FourOhFour.propTypes = {
	classes: PropTypes.object.isRequired,
};

export default withStyles(styles)(FourOhFour);
