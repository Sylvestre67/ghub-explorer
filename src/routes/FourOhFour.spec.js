import React from 'react';
import { shallow } from 'enzyme';

import { FourOhFour } from './FourOhFour';
let component;

const props = {
	classes: {},
};

describe('FourOhFour', () => {
	beforeAll(() => {
		component = shallow(<FourOhFour {...props} />);
	});

	it('should render correctly', () => {
		expect(component).toMatchSnapshot();
	});
});
