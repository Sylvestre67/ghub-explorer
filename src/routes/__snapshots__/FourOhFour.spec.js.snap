// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FourOhFour should render correctly 1`] = `
<div>
  <WithStyles(Typography)
    color="error"
    variant="h6"
  >
    Not Found
  </WithStyles(Typography)>
  <WithStyles(Typography)>
    The Page you requested cannot be found.
  </WithStyles(Typography)>
</div>
`;
